azure/table_azure_storage_container.go

```diff
@@ -0,0 +1,153 @@
+package azure
+
+import (
+	"context"
+
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+
+	"github.com/Azure/azure-sdk-for-go/services/storage/mgmt/2019-06-01/storage"
+)
+
+//// TABLE DEFINITION
+
+func tableAzureStorageContainer(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "azure_storage_container",
+		Description: "Azure Storage Container",
+		Get: &plugin.GetConfig{
+			KeyColumns:        plugin.AllColumns([]string{"name", "resource_group", "account_name"}),
+			Hydrate:           getStorageContainer,
+			ShouldIgnoreError: isNotFoundError([]string{"ResourceNotFound", "ResourceGroupNotFound", "ContainerNotFound"}),
+		},
+		List: &plugin.ListConfig{
+			ParentHydrate: listStorageAccounts,
+			Hydrate:       listStorageContainers,
+		},
+
+		Columns: []*plugin.Column{
+			// Basic info
+			{
+				Name:        "name",
+				Description: "The friendly name that identifies the container.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "id",
+				Description: "The container ID",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ID"),
+			},
+			{
+				Name:        "type",
+				Description: "Specifies the type of the container.",
+				Type:        proto.ColumnType_STRING,
+			},
+			// Other details
+			{
+				Name:        "account_name",
+				Description: "The friendly name that identifies the storage account.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ID").Transform(idToAccountName),
+			},
+			{
+				Name:        "container_properties",
+				Description: "The blob container properties.",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("ContainerProperties"),
+			},
+
+			// Standard steampipe columns
+			{
+				Name:        "title",
+				Description: ColumnDescriptionTitle,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Name"),
+			},
+			{
+				Name:        "tags",
+				Description: ColumnDescriptionTags,
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("Etag"),
+			},
+			{
+				Name:        "akas",
+				Description: ColumnDescriptionAkas,
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("ID").Transform(idToAkas),
+			},
+
+			// Standard azure columns
+			{
```

Comment:
Don't we get `location`?
If not then can we get the location from storage account?

Reply:
as discussed will not have location field